src/rgw/store/sfs/writer.cc

```diff
@@ -107,7 +107,6 @@ int SFSAtomicWriter::complete(
 ) {
   lsfs_dout(dpp, 10) << "accounted_size: " << accounted_size
                      << ", etag: " << etag
-                     << ", mtime: " << to_iso_8601(*mtime)
```

Comment:
Instead of dropping this, can't we just print: `meta.mtime` after it has been initialized?
We could move this `lsfs_dout` call after `meta` has been initialized.

Reply:
IMHO, the utility of logging all this is to log the parameters received by the method. (I could agree to print the value of the pointer, but tbh... I don't think all those logs are that important)
I dropped it because it's an output parameter.
If we move the call after the method body... (or at the middle) I think we're changing its purpose and will look weird.

Reply:
ok, so lgtm

Reply:
I think you have to fix the checklist btw

Reply:
I changed it to don't drop it.
If the pointer passed is null it will print `"null"` otherwise it will print the `mtime` passed.

Reply:
okay... dropped it again :).